dev/ory/oathkeeper_rules.yaml

```diff
@@ -23,12 +37,14 @@
         preserve_path: true
         preserve_query: true
         subject_from: identity.id
+        extra_from: identity.traits
     - handler: bearer_token
       config:
-        check_session_url: "http://e2e-tests:4012/auth/validatetoken"
+        check_session_url: "http://kratos:4433/sessions/whoami"
         preserve_path: true
         preserve_query: true
-        subject_from: sub
+        subject_from: identity.id
+        extra_from: identity.traits
```

Comment:
@nicolasburtey adding this to use kratos instead of our custom `validatetoken` endpoint. I needed to change the `subject_from` field because it was returning blank when we switched from our graphql api `auth/validatetoken` to the kratos `whoami` endpoint

Reply:
It appears this makes the github actions e2e test fail with this error 

```
header › getting a kratos header when passing a legacy JWT in the header
AxiosError: Request failed with status code 401
```
Are the legacy jwt's fully depreciated yet?

Reply:
going to revert this back because the admin panel is still using the jwts